uuid/unstable_v6.ts

```diff
@@ -0,0 +1,178 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+// This module is browser compatible.
+
+import { bytesToUuid } from "./_common.ts";
+
+const UUID_RE =
+  /^[0-9a-f]{8}-[0-9a-f]{4}-6[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
+
+/**
+ * Determines whether a string is a valid
+ * {@link https://www.rfc-editor.org/rfc/rfc9562.html#name-uuid-version-6 | UUIDv6}.
+ *
+ * @param id UUID value.
+ *
+ * @returns `true` if the string is a valid UUIDv6, otherwise `false`.
+ *
+ * @example Usage
+ * ```ts
+ * import { validate } from "@std/uuid/unstable-v6";
+ * import { assert, assertFalse } from "@std/assert";
+ *
+ * assert(validate("1efed67d-d966-6490-8b9a-755015853480"));
+ * assertFalse(validate("1efed67d-d966-1490-8b9a-755015853480"));
+ * ```
+ */
+export function validate(id: string): boolean {
+  return UUID_RE.test(id);
+}
+
+let _nodeId: number[];
+let _clockseq: number;
+
+let _lastMSecs = 0;
+let _lastNSecs = 0;
+
+/** Options for {@linkcode generate}. */
+export interface GenerateOptions {
+  /**
+   * An array of 6 bytes that represents a 48-bit IEEE 802 MAC address.
+   *
+   * @see {@link https://www.rfc-editor.org/rfc/rfc4122#section-4.1.6}
```

Comment:
Can you update the reference to rfc9562?

Reply:
Updated